features/.gitignore

```diff
@@ -0,0 +1,2 @@
+/.classpath
+/.project
```

Comment:
I would remove this file and update the .gitignore located at the root folder of this repository to ignore .classpath and .project